autonomic-plugin-common/src/main/java/br/com/autonomiccs/autonomic/plugin/common/daos/configurations/ConfigureDatabaseDao.java

```diff
@@ -82,97 +87,88 @@ protected void initDao() throws Exception {
     }
 
     /**
-     * TODO Documentation
-     *
-     * @return
+     * It returns true if has the 'administration_status' column at the 'host' table.

```

Comment:
It returns true if there is the 'administration_status' column at the 'host' table